src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -491,9 +491,9 @@ export class BrowsingContextImpl {
   ): Promise<BrowsingContext.PROTO.FindElementResult> {
     await this.#targetDefers.targetUnblocked;
 
-    const functionDeclaration = String((resultsSelector: string) =>
-      document.querySelector(resultsSelector)
-    );
+    const functionDeclaration = `
+      (resultsSelector) => document.querySelector(resultsSelector)
+    `;
```

Comment:
Did the previous version stop working? Because it allowed to provide TS with all the compilation checks.